finished last exercise; added exercises from old MTH 60 mixing section

aecary opened the pull request:
All: That last example was originally only a one variable problem. I changed it to be two variable, but it might be too contrived now.

Carl: The exercises that I added from the MTH 60 mixing section have solutions that use tables instead of systems of equations. Can you change these, or let me know where they exist if they already do?

Carl/Alex: There are two WW problems that do not appear to be working:
BasicAlgebra/SystemsOfLinearEquations/SystemOfEquations13.pg
BasicAlgebra/SystemsOfLinearEquations/SystemOfEquations14.pg

yaoxiaolong commented:
Hi Ann:

System equation application problems are in this folder instead:

BasicAlgebra/SystemOfLinearEquationsApplications/

There are all types of problems, including interest problems and mixture
problems. Let me know if I need to write more.

Also, can you let me know why these two problems are not working:

BasicAlgebra/SystemsOfLinearEquations/SystemOfEquations13.pg
BasicAlgebra/SystemsOfLinearEquations/SystemOfEquations14.pg

They are working on my end.

Finally, the first example is great. My concern is it is too long and too
complicated to be an introductory problem for this section. Image you are
an average MTH65 student. Would you read through this long example? I think
my original example serves the purpose. Maybe this one could be moved to a
later spot?

Carl Yao

On Tue, Apr 25, 2017 at 8:43 PM, aecary <notifications@github.com> wrote:

> All: That last example was originally only a one variable problem. I
> changed it to be two variable, but it might be too contrived now.
>
> Carl: The exercises that I added from the MTH 60 mixing section have
> solutions that use tables instead of systems of equations. Can you change
> these, or let me know where they exist if they already do?
>
> Carl/Alex: There are two WW problems that do not appear to be working:
> BasicAlgebra/SystemsOfLinearEquations/SystemOfEquations13.pg
> BasicAlgebra/SystemsOfLinearEquations/SystemOfEquations14.pg
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/PCCMathSAC/orcca/pull/131
> Commit Summary
>
>    - finished last exercise; added exercises from old MTH 60 mixing
>    section
>
> File Changes
>
>    - *M* src/section-solving-system-equations-by-substitution.mbx
>    <https://github.com/PCCMathSAC/orcca/pull/131/files#diff-0> (99)
>
> Patch Links:
>
>    - https://github.com/PCCMathSAC/orcca/pull/131.patch
>    - https://github.com/PCCMathSAC/orcca/pull/131.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PCCMathSAC/orcca/pull/131>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGW_oUUAwAtwgzGPOuP-5nCb61ioqjrLks5rzr10gaJpZM4NIU7L>
> .
>



-- 
Regards

Carl Yao
Math Instructor
PCC Southeast Campus

aecary commented:
Thanks, Carl. I'll look there are update them within the next few days.

-Ann

Ann Cary
Mathematics Instructor
Portland Community College
Sylvania Campus, ST 203
971-722-7590
ann.cary@pcc.edu
<http://goog_233805033>
http://spot.pcc.edu/~acary/
Pronouns: she/her/hers


On Wed, Apr 26, 2017 at 7:00 AM, Carl Yao <notifications@github.com> wrote:

> Hi Ann:
>
> System equation application problems are in this folder instead:
>
> BasicAlgebra/SystemOfLinearEquationsApplications/
>
> There are all types of problems, including interest problems and mixture
> problems. Let me know if I need to write more.
>
> Also, can you let me know why these two problems are not working:
>
> BasicAlgebra/SystemsOfLinearEquations/SystemOfEquations13.pg
> BasicAlgebra/SystemsOfLinearEquations/SystemOfEquations14.pg
>
> They are working on my end.
>
> Finally, the first example is great. My concern is it is too long and too
> complicated to be an introductory problem for this section. Image you are
> an average MTH65 student. Would you read through this long example? I think
> my original example serves the purpose. Maybe this one could be moved to a
> later spot?
>
> Carl Yao
>
> On Tue, Apr 25, 2017 at 8:43 PM, aecary <notifications@github.com> wrote:
>
> > All: That last example was originally only a one variable problem. I
> > changed it to be two variable, but it might be too contrived now.
> >
> > Carl: The exercises that I added from the MTH 60 mixing section have
> > solutions that use tables instead of systems of equations. Can you change
> > these, or let me know where they exist if they already do?
> >
> > Carl/Alex: There are two WW problems that do not appear to be working:
> > BasicAlgebra/SystemsOfLinearEquations/SystemOfEquations13.pg
> > BasicAlgebra/SystemsOfLinearEquations/SystemOfEquations14.pg
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/PCCMathSAC/orcca/pull/131
> > Commit Summary
> >
> > - finished last exercise; added exercises from old MTH 60 mixing
> > section
> >
> > File Changes
> >
> > - *M* src/section-solving-system-equations-by-substitution.mbx
> > <https://github.com/PCCMathSAC/orcca/pull/131/files#diff-0> (99)
> >
> > Patch Links:
> >
> > - https://github.com/PCCMathSAC/orcca/pull/131.patch
> > - https://github.com/PCCMathSAC/orcca/pull/131.diff
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/PCCMathSAC/orcca/pull/131>, or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/AGW_oUUAwAtwgzGPOuP-
> 5nCb61ioqjrLks5rzr10gaJpZM4NIU7L>
> > .
> >
>
>
>
> --
> Regards
>
> Carl Yao
> Math Instructor
> PCC Southeast Campus
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PCCMathSAC/orcca/pull/131#issuecomment-297417527>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AGBwA95SY9UH_IcYYuw1S0AWJDeCYpwVks5rz03igaJpZM4NIU7L>
> .
>
